Makefile

```diff
@@ -4,6 +4,9 @@ build:
 pivot:
 	@python Build/Scripts/seajax_v2.py pivot
 
+collegevine:
+	@python Build/Scripts/seajax_v2.py collegevine
```

Comment:
Quick `make collegevine` command for compiling new standalone Pivot viewer specific to CollegeVine.